stablehlo/unittests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 The StableHLO Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# StableHLO unit tests are run in a non-obvious way.
+# We cannot just say `ninja StablehloUnitTests` - that would simply build
```

Comment:
Is this based off something you saw in another project / is this the standard way of doing this? I would imagine elsewhere in openxla has this issue as well, want to make sure the solution is consistent.